lib/verifier/post_zephir_verifier.rb

```diff
@@ -0,0 +1,195 @@
+# frozen_string_literal: true
+
+require "zlib"
+require_relative "../verifier"
+require_relative "../derivatives"
+
+# Verifies that post_zephir workflow stage did what it was supposed to.
+
+# TODO: document and verify the files written by monthly process.
+# They should be mostly the same but need to be accounted for.
+
+module PostZephirProcessing
+  class PostZephirVerifier < Verifier
```

Comment:
We should consider splitting this up into smaller classes each of which represent the e.g. `verify_catalog_archive`, `verify_rights`, and then collecting up the errors similar to what we're doing in `HathifilesVerifier` for any integration testing.

Reply:
To do later: #54